metrics/internal/cache/rbd-mirror.go

```diff
@@ -167,9 +167,12 @@ func initCeph(kubeclient clientset.Interface, allowedNamespaces []string) (cephM
 	if !ok {
 		return cephMonitorConfig{}, fmt.Errorf("failed to get client key from secret in namespace %q", namespace)
 	}
-	id := "admin"
+	id, ok := secret.Data["ceph-user"]
```

Comment:
This provides id as `client.admin`, but earlier we were using `admin` only.
Does both of these work or do we need to strip the `client.` prefix? @travisn @Madhu-1

Reply:
if we have client.x we need to strip `client.` prefix for CLI commands to work or else we will get permission denied error.

Reply:
Ack. Made the change.